### ISSUE TITLE
django.urls.exceptions.NoReverseMatch when hitting the show on website button in form changeform

### ISSUE THREAD

**Issue by max-pfeiffer** [barseghyanartur/django-fobi#53]
Using Django 1.10 and Python 3.4.

```
[12/Jan/2017 17:38:47] ERROR [django.request:124] Internal Server Error: /django-admin/r/23/1/
Traceback (most recent call last):
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/core/handlers/exception.py", line 39, in inner
    response = get_response(request)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/contrib/admin/sites.py", line 229, in wrapper
    return self.admin_view(view, cacheable)(*args, **kwargs)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/utils/decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/views/decorators/cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/contrib/admin/sites.py", line 211, in inner
    return view(request, *args, **kwargs)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/contrib/contenttypes/views.py", line 31, in shortcut
    absurl = get_absolute_url()
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/fobi/models.py", line 394, in get_absolute_url
    return reverse('fobi.view_form_entry', kwargs={'slug': self.slug})
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/urls/base.py", line 91, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/urls/resolvers.py", line 392, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns)
django.urls.exceptions.NoReverseMatch: Reverse for 'fobi.view_form_entry' with arguments '()' and keyword arguments '{'slug': 'test'}' not found. 1 pattern(s) tried: ['fobi/view/(?P<form_entry_slug>[\\w_\\-]+)/$']
[12/Jan/2017 17:38:47] ERROR [django.server:131] "GET /django-admin/r/23/1/ HTTP/1.1" 500 113707
```

**Comment by barseghyanartur** [comment]
@max-pfeiffer: Regarding all django-admin related issues, you're not supposed to use Django fobi using classic django-admin. They are still bugs, though, to be fixed.

**Comment by barseghyanartur** [comment]
@max-pfeiffer: Fixed in 0.10.4.
